content/en/security/cloud_security_management/agentless_scanning/_index.md

```diff
@@ -114,9 +114,12 @@ Along with displaying sensitive data matches, Sensitive Data Scanner surfaces an
 
 ## Cloud service provider cost
 
-When using Agentless Scanning, there are additional costs for running scanners in your cloud environments. To optimize on costs while being able to reliably scan every 12 hours, Datadog recommends setting up [Agentless Scanning with Terraform][6] as the default template, as this also avoids cross-region networking.
+When using Agentless Scanning, there are additional cloud provider costs for running scanners and analyzing your cloud environments.
+
+Cloud provider costs are under $1 per scanned host per year for accounts following the [recommended configuration][13]. Reach out to your [Datadog Customer Success Manager][7] if you have more questions.
```

Comment:
We usually don't talk about costs in the docs, and usually link to the pricing page. In this situation, can we just have customers reach out to CSM about costs? So something like:

```suggestion
Reach out to your [Datadog Customer Success Manager][7] if you have questions about Cloud provider costs for the [recommended configuration][13].
```

Reply:
Hi May!
So that's the point of this change: we have too many customer requests about this cost. 
It's unrelated to the Datadog pricing: that's only about how running this Datadog feature will increase your cloud provider bill.